src/core/templates/partials/component.jsx

```diff
@@ -0,0 +1,59 @@
+import Adapt from 'core/js/adapt';
+import { compile, classes, helper, html } from 'core/js/reactHelpers';
+
+export default function(model, view) {
+  const data = model.toJSON();
+  data._globals = Adapt.course.get('_globals');
+  // Create references to un-controlled view containers
+  view.jsxHeading = view.jsxHeading || React.createRef();
+  view.jsxComponentDescription = view.jsxComponentDescription || React.createRef();
+  const {
+    displayTitle,
+    body,
+    instruction,
+    mobileInstruction,
+    _component,
+    _disableAccessibilityState
+  } = data;
+  const type = _component.toLowerCase();
+  const sizedInstruction = (mobileInstruction && Adapt.device.screenSize !== 'large') ?
+    mobileInstruction :
+    instruction;
+  return (displayTitle || body || sizedInstruction) &&
+    <div className={classes(['component__header', `${type}__header`])}>
+      <div className={classes(['component__header-inner', `${type}__header-inner`])}>
+        {displayTitle &&
+        <div className={classes(['component__title', `${type}__title`])}>
+
+          {!_disableAccessibilityState &&
+          <div className="js-heading" ref={view.jsxHeading}></div>
+          }
+
+          <div className={classes(['component__title-inner', `${type}__title-inner`])} aria-hidden={!_disableAccessibilityState}>
+            {`\n${html(compile(displayTitle, data))}\n`}
```

Comment:
What's the reasoning behind the manual newline characters?

Reply:
Styling is currently reliant on the whitespace, the line collapses in an odd way without it.

Reply:
Okay, let's not rely on the quirkiness of yesteryear. Please could you remove the newline characters from the JSX and provide a level playing field by changing this:
https://github.com/adaptlearning/adapt_framework/blob/befc898ba0e17b58f81cc9ce7aaaf8c770864a87/src/core/templates/partials/component.hbs#L16
to:
```hbs
{{~{compile displayTitle}~}}
```
If a slight margin needs to be added then so be it (have discussed this briefly with @guywillis).

Reply:
Body and instruction also?